src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
```

Comment:
**suggestion**: Consider adding PropTypes or TypeScript interfaces to define the expected props for this component. This will enhance type safety and improve code readability.